test/evergreen.yml

```diff
@@ -2821,207 +2822,33 @@ tasks:
-            ${python_binary|python3} ../test/suite/run.py test_isolation01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_join08 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_jsondump01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_log03 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_lsm04 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_metadata_cursor02 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_overwrite 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_pack 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_prepare_cursor01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_prepare_hs03 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_prepare15 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_readonly03 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_reconfig03 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_rename 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_reserve 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_salvage01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_schema07 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_search_near02 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_shared_cache01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_split 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_stat_log01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_stat06 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_sweep04 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_tiered08 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_timestamp10 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_truncate15 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_turtle01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_txn15 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_unicode01 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_upgrade 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_util15 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_verbose02 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_verify2 2>&1
-            ${python_binary|python3} ../test/suite/run.py test_version 2>&1
             # Record the end time, in seconds
-            date +%s >> ../time.txt
+            date +%s >> time.txt
       - func: "code coverage analysis"
         vars:
           generate_atlas_format: true
+          working_dir: "wiredtiger/build_0"
```

Comment:
It finds it because the invocation of `GCOVR` refers to the above directory (ie `wiredtiger` though the `-r ..` parameter), and then `GCOVR` finds all the code coverage files in any sub-directories (so any of `build_0`, `build_1`, etc). So actually it could be run in any of the build directories and would still work.

I'll have another look into making this more obvious and/or adding a comment.

Reply:
And add more info the README.md file.

Reply:
Done!